app/server.go

```diff
@@ -746,6 +747,8 @@ func (s *Server) Shutdown() {
 		}
 	}
 
+	s.pluginService.ShutDownPlugins()
```

Comment:
I think plugins should be shutdown before products.  A product won't depend on a plugin, but plugin will depend on products.

Reply:
Makes sense 👍